frontend/src/App.vue

```diff
@@ -106,6 +99,13 @@
             </p>
           </div>
         </div>
+        <div class="column has-text-centered">
+          <p>2021 ©
+            <span class="is-hidden-touch">
```

Comment:
```suggestion
            <span class="is-hidden-mobile">
```

Do you think this is more appropriate? Here is a list of the `is-hidden-X` helpers: https://bulma.io/documentation/helpers/visibility-helpers/#hide. If you intended to hide it for both mobile and tablet please ignore this.